### DISCUSSION TITLE
show and tell picture

### DISCUSSION THREAD

**Discussion by sjmorikawa** [inkstitch/inkstitch#1005] [Show and tell]
I love this app- I am new to embroidery.  Did a Hatch trial, loved it but so $$$$.  Thank you to everyone who has worked on it.

I digitized an illustration from Pixabay- not perfect, but I learned a lot!
![20210129_125516](https://user-images.githubusercontent.com/75394978/106310782-19764400-6232-11eb-93fb-63b805c53925.jpg)

**Comment by lexelby** [comment]
Awesome!  I love the way you used loose white fill stitch for the windows.
